appinventor/blocklyeditor/src/field_lexical_variable.js

```diff
@@ -323,8 +323,20 @@ Blockly.FieldLexicalVariable.getLexicalNamesInScope = function (block) {
             rememberName(loopName2, loopNames, Blockly.loopParameterPrefix);
           } else if ( (parent.type === "lists_sort_key")
                   && (parent.getInputTargetBlock('KEY') == child)) {// Only DO is in scope, not other inputs!
-           var loopName = parent.getFieldValue('VAR');
-           rememberName(loopName, loopNames, Blockly.loopParameterPrefix);
+            var loopName = parent.getFieldValue('VAR');
+            rememberName(loopName, loopNames, Blockly.loopParameterPrefix);
+          } else if ( (parent.type === "lists_minimum_number")
+                  && (parent.getInputTargetBlock('COMPARE') == child)) {// Only DO is in scope, not other inputs!
```

Comment:
Code style: Indentation.

Also, I'm not sure the comment on this line is correct.